### ISSUE TITLE
Training set and aggregation method

### ISSUE THREAD

**Issue by wincj** [amaralibey/Bag-of-Queries#29]
Hello author, this is an excellent job and I would like to ask you one questions
def forward(self, x):
# reduce input dimension using 3x3 conv when using ResNet
x = self.proj_c(x)
x = x.flatten(2).permute(0, 2, 1)
x = self.norm_input(x)

outs = []
attns = []
for i in range(len(self.boqs)):
x, out, attn = self.boqs[i](x)
outs.append(out)
attns.append(attn)
﻿
out = torch.cat(outs, dim=1)
out = self.fc(out.permute(0, 2, 1))
out = out.flatten(1)
out = torch.nn.functional.normalize(out, p=2, dim=-1)
return out， attns
﻿
﻿May I ask if BoQ returns "attns", where "attns" are only used for visualization? If I want to directly migrate your aggregation method to other code, I found that other code only returns global feature vectors at the end, so can I directly delete the last returned attns
Only      return out
